### ISSUE TITLE
AST Explorer

### ISSUE THREAD

**Issue by marcustisater** [postcss/postcss.com#20]
AST explorer to be embedded/integrated for plugin developers. There is a [issue]((https://github.com/fkling/astexplorer/issues/70) that has to be resolved first. Feel free to help.

@fkling 
> I started working on something that lets you specify the parser in the URL. That should be good enough for a start I guess.

**Comment by mxstbr** [comment]
Is this planned for launch? /cc @ai

**Comment by ai** [comment]
Current AST Explorer can be embed only by iframe. Maybe we should just put a link.

**Comment by mxstbr** [comment]
Agreed, that might be a good plan, and try to get it standalone for v1.1.

Pinging @fkling, do you think you can get it ready for v1 of postcss.com?

**Comment by marcustisater** [comment]
It's planned to be released for postcss.com 1.0 but if it's not ready by then an hyperlink should be fine.

**Comment by marcustisater** [comment]
Ticket #11 

@fkling 
> I started working on something that lets you specify the parser in the URL. That should be good enough for a start I guess.

**Comment by fkling** [comment]
When do you want to release 1.0?

I will focus on an useful iframe integration first. Not sure yet how a a true standalone version would look like...

**Comment by marcustisater** [comment]
iframe integration is great, we have no release date yet for 1.0 but hopefully before the end of Mar.

**Comment by fkling** [comment]
Oh, OK. So I don't have to be finish this today :D

**Comment by marcustisater** [comment]
Any progress @fkling?
